cronjobs_cern/indico_cronjobs_cern/plugin.py

```diff
@@ -33,6 +38,7 @@ class CERNCronjobsPlugin(IndicoPlugin):
     configurable = True
     settings_form = SettingsForm
     default_settings = {
+        'sender_email': '',
```

Comment:
since it's a CERN plugin i'd keep our default CERN address here instead of leaving it empty